--- gtk-vnc-0.5.0/plugin/npunix.c~	2011-12-21 18:07:37.000000000 +0100
+++ gtk-vnc-0.5.0/plugin/npunix.c	2012-05-04 05:30:24.738525926 +0200
@@ -393,11 +393,11 @@
  *  - Netscape uses the return value to identify when an object instance
  *    of this plugin should be created.
  */
-char *
+/*char *
 NP_GetMIMEDescription(void)
 {
     return NPP_GetMIMEDescription();
-}
+}*/
 
 /*
  * NP_GetValue [optional]
--- gtk-vnc-0.5.0/plugin/npshell.c~	2011-12-21 17:07:37.000000000 +0000
+++ gtk-vnc-0.5.0/plugin/npshell.c	2012-05-04 03:46:20.790013905 +0000
@@ -91,11 +91,11 @@
  *
  ***********************************************************************/
 
-char *
+/*char *
 NPP_GetMIMEDescription(void)
 {
   return (char *) MIME_TYPES_HANDLED;
-}
+}*/
 
 NPError
 NPP_GetValue(NPP instance G_GNUC_UNUSED, NPPVariable variable, void *value)
